lnbits/tasks.py

```diff
@@ -149,37 +148,36 @@ async def check_pending_payments():
     incoming = True
 
     while True:
-        async with db.connect() as conn:
-            logger.info(
-                f"Task: checking all pending payments (incoming={incoming},"
-                f" outgoing={outgoing}) of last 15 days"
-            )
-            start_time = time.time()
-            pending_payments = await get_payments(
-                since=(int(time.time()) - 60 * 60 * 24 * 15),  # 15 days ago
-                complete=False,
-                pending=True,
-                outgoing=outgoing,
-                incoming=incoming,
-                exclude_uncheckable=True,
-                conn=conn,
-            )
-            for payment in pending_payments:
-                await payment.check_status(conn=conn)
+        logger.info(
+            f"Task: checking all pending payments (incoming={incoming},"
+            f" outgoing={outgoing}) of last 15 days"
+        )
+        start_time = time.time()
+        pending_payments = await get_payments(
+            since=(int(time.time()) - 60 * 60 * 24 * 15),  # 15 days ago
+            complete=False,
```

Comment:
- `completed` is `False` by default

Reply:
> * `completed` is `False` by default

[explicit > implicit](https://peps.python.org/pep-0020/)